src/Utils/CMakeLists.txt

```diff
@@ -27,5 +30,23 @@ install(
   DESTINATION include
   )
 
+if(HIOP_USE_CUDA)
+  set(hiopUtils_CUDA_SRC
+    MathCudaKernels.cu
+  )
+  set_source_files_properties(${hiopUtils_CUDA_SRC} PROPERTIES LANGUAGE CUDA)
+  list(APPEND hiopUtils_SRC ${hiopUtils_CUDA_SRC})
+elseif(HIOP_USE_HIP)
+  set(hiopUtils_HIP_SRC
+    MathHipKernels.cpp
+  )
+# Mark source files as HIP. I guess in the future just a
```

Comment:
indentation, but I will fix myself when fixing expected merge conflicts for PR#510